h/features/models.py

```diff
@@ -105,3 +107,32 @@ def remove_old_flags(cls):
 
     def __repr__(self):
         return '<Feature {f.name} everyone={f.everyone}>'.format(f=self)
+
+
+class FeatureCohort(Base, mixins.Timestamps):
+    __tablename__ = 'featurecohort'
+
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+    name = sa.Column(sa.UnicodeText(), nullable=False, index=True)
+
+    # Cohort membership
+    members = sa.orm.relationship(
+        'User', secondary='featurecohort_user', backref=sa.orm.backref(
+            'cohorts', order_by='FeatureCohort.name'))
```

Comment:
The `order_by` probably isn't necessary so this can just become `backref='cohorts'`